api/lib/express.ts

```diff
@@ -64,3 +66,18 @@ export function representation (req: Request, res: Response, next: NextFunction)
 
   next()
 }
+
+export const authentication = jwt({
+  secret: jwksRsa.expressJwtSecret({
+    cache: true,
+    rateLimit: true,
+    jwksRequestsPerMinute: 5,
+    jwksUri: env.AUTH_JWKS_URI,
+  }),
+
+  // Validate the audience and the issuer.
+  audience: env.AUTH_AUDIENCE,
+  issuer: env.AUTH_ISSUER,
+  algorithms: ['RS256'],
+  credentialsRequired: true,
+})
```

Comment:
Maybe make that a noop if the auth env variables are not setup?

Reply:
And leave the API accidentally unsecured? 🤔